docs/source/dev_manual/ci_cd_setup/index.rst

```diff
@@ -0,0 +1,108 @@
+
+CI/CD setup
+------------
+
+OpenFPGA implements CI/CD system using Github actions.
+The following figure shows the Actions implements flow.
+The source building is skipped if there are changes only in ``openfpga_flow`` or ``docs`` directory,
```

Comment:
``openfpga_flow`` may be changed a lot when updating existing tests. It is very very risky to bypass the regression tests!!!

Reply:
Its doesn't not skip `regression tests` it skips the build `". . .source building is skipped. . ."`
Regression tests are always run doesn't matter which action triggers it

Reply:
Got it. It is safe then.